tests/ci_build/rename_whl.py

```diff
@@ -0,0 +1,45 @@
+import sys
+import os
+from contextlib import contextmanager
+
+def normpath(path):
```

Comment:
Can this be done by https://docs.python.org/3/library/os.path.html#os.path.normpath ?